tests/console/login.go

```diff
@@ -117,10 +130,11 @@ func LoginToFedora(vmi *v1.VirtualMachineInstance) error {
 	b := append([]expect.Batcher{
 		&expect.BSnd{S: "\n"},
 		&expect.BExp{R: PromptExpression},
+		&expect.BSnd{S: "true\n"},
```

Comment:
I don't understand why the `true` command is needed here. Couldn't understand the explanation in the commit message as well.

Reply:
Lets say one sends command that fails, but its output isn't consumed 
then the relogin will find the PromptExpression but the echo $? will failed and the flow will continue the login process below (it will find the prompt there, so it won't send user:pass again, but it will reconfigure again the shell, and the whole thing is slower)

site node - this can be improved on other PR to match the full prompt (localhost|%s) login: `, vmi.Name)
instead of PromptExpression
because PromptExpression can be even matched during boot when the machine isn't actually logged
see here for example test 1705 when i totally removed the 
```
		&expect.BSnd{S: "echo $?\n"},
		&expect.BExp{R: RetValue("0")},
```
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/6116/pull-kubevirt-e2e-k8s-1.20-sig-network/1422158507934748672
Then it failed because it didn't need more condition after it matched the `# `
the `echo $?` makes it better but still not perfect but not need to be part of this PR

even adding the true can be done on other PR if needed basically, but it does improve the login in case the last command fail and its output was not consumed (for example timeout, or if ran manually and then ran `make dump`)